.github/workflows/snyk.yml

```diff
@@ -1,39 +1,18 @@
-# This action runs Snyk Monitor every day at 6 AM and on every push to main
+# This action runs snyk monitor on every push to main
 name: Snyk
 
 on:
-  schedule:
-    - cron: '0 6 * * *'
   push:
     branches:
-      - master
+      - main
```

Comment:
We need to move the default branch to main but I'll do that before this is merged